user-interface/src/components/CaseAssignment.tsx

```diff
@@ -72,12 +72,10 @@ export const CaseAssignment = () => {
           // Filing date formatted in SQL query as MM-dd-yyyy (ISO is yyyy-MM-dd)
           // dateFiled[0] = month, dateFiled[1] = day, dateFiled[2] = year
           caseNode.sortableDateFiled = `${dateFiled[2]}${dateFiled[0]}${dateFiled[1]}`;
-          if (caseNode.assignments) {
-            if (caseNode.assignments.length > 0) {
-              assignmentList.push(caseNode);
-            } else {
-              nonAssignmentList.push(caseNode);
-            }
+          if (caseNode.assignments && caseNode.assignments.length > 0) {
+            assignmentList.push(caseNode);
+          } else {
+            nonAssignmentList.push(caseNode);
```

Comment:
This change allows us to cover the condition where `caseNode.assignments` is falsy, where before we were not covering that condition. Semantically speaking this condition should not happen because the backend sends an empty array, but that isn't really a business requirement, just something we do.